test/test.js

```diff
@@ -1,34 +1,46 @@
-var expect = require('chai').expect;
-var supertest = require('supertest');
-var server = require('../index').server;
-var utils = require('../utils');
-
-var expectJSON = function(app, path){
-  return function(){
-    return app.get(path).expect('Content-Type', /json/);
-  };
-};
-
-describe('Something', function(){
-  it('should do something', function(){
-    return expect({a:1}).to.not.have.property('b');
-  });
-});
-
-describe('API routes', function(){
-  var api = supertest(server);
-
-  describe('GET /version', function(){
+import { expect } from 'chai'
+import supertest from 'supertest'
+import { server } from '../src/index'
+import utils from '../src/utils'
+
+const expectJSON = (app, path) => {
+  return () => {
+    return app.get(path).expect('Content-Type', /json/)
+  }
+}
+
+describe('Something', () => {
+  it('should do something', () => {
+    return expect({a:1}).to.not.have.property('b')
+  })
+})
+
+// we need to fix this somehow
+describe('API routes', () => {
+  let api
+
+  beforeEach(() => {
+    api = supertest(server)
+  })
+
+  describe('GET /version', () => {
     // this beforeEach is needed as SuperAgent doesn't like .end() called more than once
-    var path;
-    beforeEach(function(){path = api.get('/version');});
-
-    it('responds with package version', function(){
-      return path.expect(200, {version: process.env.npm_package_version});
-    });
-
-    it('responds with json', function(){
-      return path.expect('Content-Type', /json/);
-    });
-  });
-});
+    let path
```

Comment:
you misnamed the variables here. `var api = supertest(server)` and `api.get('/verson')` doesn't exist.